selfhost/error.jakt

```diff
@@ -172,6 +173,35 @@ fn print_source_line(severity: MessageSeverity, file_contents: [u8], file_span:
     eprintln("")
 }
 
+fn print_underline(severity: MessageSeverity, width: usize, file_span: (usize, usize), error_span: Span, line_number: usize, largest_line_number: usize) throws {
+    mut index = file_span.0
+
+    // Print the error label
+    for x in 0..(width + 2) {
+        eprint(" ")
+    }
+    eprint("│ ")
+
+    while index <= file_span.1 {
```

Comment:
somewhat strange not using a for loop for this seeing one above…! 😝

(itʼd be even nice to have it working with `for index in file_span`)

Reply:
Are you saying you want to to add a feature to the language to iterate over a 2-element tuple as a (start,end) pair?

Reply:
Oh I think I get what you're saying. You have two different ideas, and the first one is using a range? Yeah, could probably rewrite it to do something like that

Reply:
Fixed - the tuple idea could be worth a brainstorm, too, for some future updates

Reply:
oh! mixed things too much—happy the message got through! 😝